automated install/basic-install.sh

```diff
@@ -66,8 +66,14 @@ Cloudflare (DNSSEC);2606:4700:4700::1111;2606:4700:4700::1001
 EOM
 )
 
+# This directory is where the Pi-hole scripts will be installed
+PI_HOLE_INSTALL_DIR="/opt/pihole"
+PI_HOLE_CONFIG_DIR="/etc/pihole"
+PI_HOLE_LOCAL_REPO="/etc/.pihole"
+PI_HOLE_BIN_DIR="/usr/local/bin"
+
 # Location for final installation log storage
-installLogLoc="/etc/pihole/install.log"
+installLogLoc="${PI_HOLE_CONFIG_DIR}/install.log"
```

Comment:
I'm going to be a pain in the arse, but... please can you split these changes out to a seperate PR?

Not just the variable substitutions, but also in places where you have swapped verbose switches for the shorthand version. Although I do applaud the sorting out of the variable subs, it's good things more consistent!

I'd like this PR to be _purely_ about adding Alpine support - the less stylistic changes, the easier it is to review (or postmortem) from that perspective